README.md

```diff
@@ -55,7 +55,7 @@ This is by design and was chosen as the strategy to allow smoother migration fro
 
 The latest stable versions from these branches are:
 
-* [2.13.3](../../wiki/Jackson-Release-2.13), released on 14-May-2022
+* [2.13.4](../../wiki/Jackson-Release-2.13), released on 3-Sept-2022
     * [2.12.6](../../wiki/Jackson-Release-2.12) is the latest patch from previous stable branch
```

Comment:
https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.12.7 is latest 2.12 release

Reply:
Fair, not related to just-released version, but updated anyway 1e937b5b77380cac55504e5ba5886a77cc118a60